tests/installation/install_and_reboot.pm

```diff
@@ -57,35 +57,10 @@ sub run() {
     select_console 'installation';
     assert_screen 'rebootnow';
 
-    if (get_var("LIVECD")) {
+    # LiveCD needs confirmation for reboot
+    if   (get_var("LIVECD")) { send_key $cmd{"rebootnow"}; }
```

Comment:
please follow the style used everywhere else and put if and else on their own lines